file-service-api/src/test/java/uk/gov/justice/services/fileservice/domain/FileReferenceTest.java

```diff
@@ -21,7 +21,7 @@ public void shouldAutoCloseTheFileInputStream() throws Exception {
         final JsonObject metadata = mock(JsonObject.class);
         final InputStream contentStream = mock(InputStream.class);
 
-        try(final FileReference ignored = new FileReference(fileId, metadata, contentStream)) {
+        try(final FileReference ignored = new FileReference(fileId, metadata, contentStream, false)) {
```

Comment:
create a second constructor that commits the deleted flag and defaults to false.. this(...,..,.., false)